src/aioftp/client.py

```diff
@@ -983,7 +1055,14 @@ def append_stream(self, destination, *, offset=0):
             offset=offset,
         )
 
-    async def upload(self, source, destination="", *, write_into=False, block_size=DEFAULT_BLOCK_SIZE):
+    async def upload(
+        self,
+        source,
+        destination="",
+        *,
+        write_into=False,
+        block_size=DEFAULT_BLOCK_SIZE,
+    ):
```

Comment:
No need to reformat things, if linters are ok. The project moved from 88 to 120 line length.